#! Param 1
mass_arm: [20,0,0,
           0,20,0,
           0,0,20]
damping_arm: [50,0,0,
              0,50,0,
              0,0,50]
stiffness_coupling: [100,0,0,
                     0,100,0,
                     0,0,100]

arm_max_vel: 1.5
arm_max_acc: 1.0
platform_max_vel: 0.1
platform_max_acc: 0.2

init_pose: [0.3, 0.3, 0.1,  0, 0.707, 0.707, 0]
# init_pose: [0, 0, 0,  0, 0, 0, 0]

base_link: "base_link"
end_link: "ft_sensor"